rfc#00003

```diff
@@ -0,0 +1,213 @@
+# Title (Ex. 00002: Trust domains and Enarx)
+- Authors: [Mike Bursell](you@github-email)
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2020-02-13
+- Status Note: under discussion
+- Start Date: 2020-02-13 (date you started working on this idea)
+- Tags: trust
+- Addresses: (Issue #192)[https://github.com/enarx/enarx/issues/192]
+
+## Summary
+
+Trust domains are a key concept within Enarx.  This RFC provides an
+introduction to trust domains in Enarx.
+
+## Motivation
+
+In order to explain the motivations behind Enarx, our design principles
+and aims, we need to be able to explain what trust domains are.  This RFC
+attempts to provide an introduction.
+
+## Tutorial
+
+See rfc#00002 for an introduction to trust relationships.
+
+### Applying trust relationships to Enarx
+The key trust relationship for Enarx is that held by the tenant
+(workload owner) to that workload.  Enarx aims to provide assurances to
+the tenant that the integrity and confidentiality of that workload are
+protected: these are the key contexts of that trust, and actions can be
+assured if they are maintained.  Trust may decay or be built up in the
+workload - the time context.  In this case, the workload does not have an
+equivalent set of assurances about the tenant, so the relationship is
+not symmetrical.
+
+See rfc#00002 for an introduction to trust relationships.
```

Comment:
an intro to trust *relationships* or trust *domains*?

Reply:
fixed.